packages/neuron-ui/src/locales/zh-tw.json

```
@@ -908,7 +908,9 @@
         "copy-address": "復製地址",
         "m-n": {
           "title": "請選擇 m-of-n 類型的多重簽名地址配置，從 n 個密鑰中通過 m 個確認來轉移資產。",
-          "error": "對於 m-of-n 多重簽名地址，m 不應大於 n"
+          "m-less-equal-n": "對於 m-of-n 多重簽名地址，m 不應大於 n",
+          "m-n-required": "請輸入/選擇 m/n",
```

```suggestion
          "m-n-required": "請輸入或選擇 m/n",
```

The first `/` means `or` while the second one means `and`, a little unclear